src/compute-pr-actions.ts

```diff
@@ -138,18 +138,18 @@ function extendPrInfo(info: PrInfo): ExtendedPrInfo {
     const authorIsOwner = info.pkgInfo.every(p => p.owners.some(isAuthor));
     const editsInfra = info.pkgInfo.some(p => p.name === null);
     const editsConfig = info.pkgInfo.some(p => p.files.some(f => f.kind === "package-meta"));
-    const allOwners = unique(flatten(info.pkgInfo.map(p => p.owners)));
-    const otherOwners = allOwners.filter(o => !isAuthor(o));
-    const noOtherOwners = allOwners.every(isAuthor);
-    const tooManyOwners = allOwners.length > 50;
-    const editsOwners = info.pkgInfo.some(p => p.kind === "edit" && p.addedOwners.length + p.deletedOwners.length > 0);
     const packages = noNullish(info.pkgInfo.map(p => p.name));
     const hasMultiplePackages = packages.length > 1;
     const hasDefinitions = info.pkgInfo.some(p => p.files.some(f => f.kind === "definition"));
     const hasTests = info.pkgInfo.some(p => p.files.some(f => f.kind === "test"));
     const isUntested = hasDefinitions && !hasTests;
     const newPackages = noNullish(info.pkgInfo.map(p => p.kind === "add" ? p.name : null));
     const hasNewPackages = newPackages.length > 0;
+    const allOwners = unique(flatten(info.pkgInfo.map(p => p.owners)));
+    const otherOwners = allOwners.filter(o => !isAuthor(o));
+    const noOtherOwners = packages.length > newPackages.length && otherOwners.length === 0;

```

Comment:
I really dislike this for two reasons:
1. It's an obscure way of saying that "there *are* packages that were edited" (i.e., excluding infra which are not packages).
2. It complicates the meaning of `noOtherOwners` which *seems* like it's fine in this change, but I'm not sure that this will continue hold -- it is especially suspicious in that it can confuse future authors & readers.  (See below for an existing such case, which of course you edited since you now know the change in meaning.)